mlflow/models/evaluation/base.py

```diff
@@ -667,21 +667,41 @@ def evaluate(
 
     :param model: A pyfunc model instance, or a URI referring to such a model.
 
+    :param data: One of the following:
+     - A numpy array or list of evaluation features, excluding labels.
+     - A Pandas DataFrame, or a spark DataFrame,
+       containing evaluation features and labels. If `feature_names` argument not specified,
+       all columns will be regarded as feature columns, otherwise column names which match
+       `feature_names` will be regarded as feature columns.
+
     :param model_type: A string describing the model type. The default evaluator
                        supports "regressor" and "classifier" as model types.
-    :param dataset: An instance of :py:class:`mlflow.models.evaluation.EvaluationDataset`
-                    containing features labels (optional) for model evaluation.
-    :param run_id: The ID of the MLflow Run to which to log results. If
-                   unspecified, behavior depends on the specified `evaluator`.
-                   When `run_id` is unspecified, the default evaluator logs
-                   results to the current active run, creating a new active run if
-                   one does not exist.
+
+    :param targets: (Optional) A numpy array or list of evaluation labels, if `data` is also a
+      numpy array or list.
+
+    :param label_col: (Optional) The string name of a column from `data` that contains
+      evaluation labels, if `data` is a DataFrame.
```

Comment:
Can we fix the indentation for these parameter docs? The second line should start after the colon from the first line, for example:

```suggestion
    :param targets: (Optional) A numpy array or list of evaluation labels, if `data` is also a
      numpy array or list.

    :param label_col: (Optional) The string name of a column from `data` that contains
                      evaluation labels, if `data` is a DataFrame.
```